tools/install-dependencies.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
 set -eo pipefail
+export $(cat ./tools/custom-action/.env | xargs) # Source the .env file
 
 code=1
 for opt in "$@"; do
     case "$opt" in
         bikeshed)
-            pip3 install bikeshed==3.13.1
+            pip3 install bikeshed
```

Comment:
This won't reinstall if there's already a version of bikeshed installed. Use:
```suggestion
            # Always use the latest bikeshed because that's what spec-prod uses.
            pip3 install --upgrade bikeshed
```

Reply:
This is great!